contracts/0.4.24/Lido.sol

```diff
@@ -1096,19 +884,39 @@ contract Lido is Versioned, StETHPermit, AragonApp {
         uint256 clValidators = CL_VALIDATORS_POSITION.getStorageUint256();
         // clValidators can never be less than deposited ones.
         assert(depositedValidators >= clValidators);
+
         return (depositedValidators - clValidators).mul(DEPOSIT_SIZE);
     }
 
+    /**
+     * @dev Gets the maximum allowed external balance as a percentage of total pooled ether
+     * @return max external balance in wei
+     */
+    function _getMaxExternalBalance() internal view returns (uint256) {
+        return _getTotalPooledEther().mul(MAX_EXTERNAL_BALANCE_POSITION.getStorageUint256()).div(TOTAL_BASIS_POINTS);
```

Comment:
Not sure that we want to calculate this against the sum of Lido V2 TVL and vaults TVL together

Reply:
Fixed in f560380